wpk/run.sh

```diff
@@ -2,36 +2,141 @@
 
 DIRECTORY="wazuh"
 REPOSITORY="https://github.com/wazuh/wazuh.git"
-BRANCH=${1}
-JOBS=${2}
-OUT_NAME=${3}
-CHECKSUM=${4}
-INSTALLATION_PATH=${5}
-PKG_NAME=${6}
+REFERENCE=""
+JOBS="4"
+OUT_NAME=""
+CHECKSUM="no"
+INSTALLATION_PATH="/var"
```

Comment:
Can it be changed to  `/var/ossec`?

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/467/commits/1c82879d25119ef9aeb3808e1a4f12747d8edbf0